### PR TITLE
fixes #14901 - add service files for core

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: deploy/smart_proxy_dynflow_core.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Foreman smart proxy dynflow core service
+Documentation=https://github.com/theforeman/smart_proxy_dynflow
+After=network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
```
Comment (adamruzicka) [inline deploy/smart_proxy_dynflow_core.service]: Shouldn't this be `Type=simple` since the core doesn't fork into background?

Comment (stbenjam) [inline deploy/smart_proxy_dynflow_core.service]: Oh yea, thanks

**Comment by stbenjam** [comment]
Updated

**Comment by adamruzicka** [comment]
Merging, thank you

**Pull request merged**